src/react-apps/applications/service-development/src/features/administration/components/Administration.tsx

```diff
@@ -24,13 +24,17 @@ export interface IAdministrationComponentProps extends IAdministrationComponentP
   initialCommit: ICommit;
   serviceDescription: string;
   serviceDescriptionIsSaving: boolean;
+  serviceId: string;
+  serviceIdIsSaving: boolean;
 }
 
 export interface IAdministrationComponentState {
   serviceDescription: string;
   serviceName: string;
+  serviceId: string;
   editServiceName: boolean;
   editServiceDescription: boolean;
+  editServiceId: boolean;
```

Comment:
Suggestion: Alphabetize interface names.

Reply:
Everything is now sorted 🔢